src/cli/lambdapi.ml

```diff
@@ -79,6 +79,7 @@ let decision_tree_cmd : Config.t -> (Path.t * string) -> unit =
     let sym =
       let sign = Compile.compile false mp in
       let ss = Sig_state.of_sign sign in
+      let ss = Sig_state.open_sign ss sign in
```

Comment:
Below, thanks to write `try Sig_state.find_sym ~prt:true ~prv:true false ss (Pos.none ([], sym))` as it is done in other files.

Reply:
The point was that you were searching a non-qualified symbol from a non-open module. To avoid calling Sig_state.open_sign, you could instead search for a qualified symbol, that is `(mp, sym)` instead of `([],sym)`.

Reply:
Hmm right, but the `mp` doesn't have the adequate type,
```
Error: This expression has type (Path.t * string) Pos.loc
       but an expression was expected of type
         Syntax.qident = (Syntax.p_module_path * string) Pos.loc
       Type Path.t = string list is not compatible with type
         Syntax.p_module_path = (string * bool) list 
       Type string is not compatible with type string * bool
```
so is there something fishy that should be corrected, or opening the module is the way to go? If I understand well, when we create a signature state from a signature, no signature is opened by default? (except the ghost ones)

Reply:
Right.

Reply:
What do you think of replacing `mp` by `List.map (fun s -> (s,false)) mp` instead?

Reply:
Right, it's what I've done in 826ae97 in fact.

Reply:
Sorry, I misunderstood, it wasn't what I did in 826ae97, but now it's done.